manifests/app.pp

```diff
@@ -44,7 +44,6 @@
         group   => $gid,
         mode    => '0644',
         content => template($template),
-        require => Package[$::uwsgi::package_name],
-        notify  => Service[$::uwsgi::service_name]
```

Comment:
Is this `notify` not required? Is the directory monitored by the running service?